src/components/Personal.tsx

```diff
@@ -0,0 +1,75 @@
+import { Fragment, useState } from "react";
+import style from "../components/style//Personal.module.css";
+import { TextField, Typography } from "@mui/material";
+
+let personalData = {};
+
+function Personal(props: any) {
+  const [form, setForm] = useState({});
+
+  function handleForm(e: React.ChangeEvent<HTMLInputElement>, key: string) {
+    setForm({
+      ...form,
+      [key]: e.target.value,
+    });
+  }
+  personalData = {
+    ...form,
+  };
+  props.onSavePersonalData(personalData);
+
+  return (
+    <Fragment>
+      <div className={style.form}>
+        <div className={style.personalContainer}>
+          <Typography className={style.h1}>Personal Data</Typography>
+          <TextField
+            className={style.inputContainer}
+            onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+              handleForm(e, "nameSurname")
+            }
+            type="text"
+            id="outlined-basic"
+            label="Name and surname"
+            variant="outlined"
+            color="success"
+          />
+          <TextField
+            className={style.inputContainer}
+            onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
```

Comment:
To keep the code clean I would suggest extracting this function to a separate function like handleOnChange.